wwwapp/models.py

```diff
@@ -90,28 +90,35 @@ class Meta:
     Returns a date class instance or None if the PESEL is malformed or not present.
     """
     def get_birth_date(self) -> date or None:
+        if not self.pesel.isdigit():
```

Comment:
Validity checks should be moved to the special `clean` method ([docs]).(https://docs.djangoproject.com/en/1.11/ref/models/instances/#django.db.models.Model.clean) This way they will be imposed when new data is imputed into the system. I think it's ok to calculate birthdate on invalid PESELs already in the system.

Reply:
All comments are of course valid, but the check itself was imposed nonetheless, so this is not a logical change. I will fix the code later today.

Reply:
Well, I would argue check on submitting PESEL is much more useful, then a check on presenting submitted data to organizers.